concepts/05 Widgets/Calendar/05 Handle the Value Change Event.md

```diff
@@ -39,6 +37,48 @@ To process a new calendar value, you need to handle the value change event. If t
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template> 
+        <DxCalendar @value-changed="calendarValueChanged" />
```

Comment:
It looks like in Angular and Vue we should show two-way binding and in React state changing.

Reply:
Is it actual for the **"Handle the Value Change Event"** topic?
During this event we can not only synchronize the values[two-way binding], but also perform other actions - load the data source, call specific function etc.

Reply:
Can we have both two-way binding and the valueChanged event?

Reply:
Yep. Topic updated